source/distributed/tria.cc

```diff
@@ -933,19 +933,14 @@ namespace
     return *this_object->current_pointer++;
   }
 
-
-
   template <int dim, int spacedim>
-  using quadrant_cell_relation_t = typename std::tuple<
-    typename dealii::internal::p4est::types<dim>::quadrant *,
```

Comment:
I find it interesting that we do not need the mapping from `p4est` quadrant to `dealii` cell anywhere in the code. Well, the testsuite passes so I guess this really was just a relic :)